src/lib/helpers/activityCategorization.ts

```diff
@@ -10,30 +10,52 @@ const activityCategorization: RezervoCategory[] = [
         color: "#FFFF66",
         keywords: ["happening", "event"],
     },
+    {
+        name: "Tileggstjenester",
+        color: "#f4eded",
+        keywords: ["barnepass", "kroppsanalyse"],
+    },
+    {
+        name: "Vann",
```

Comment:
Is this swimming? "Vann" is not very descriptive :upside_down_face:

Reply:
Vannaerobic